llvm/test/Feature/Repo/repo-opt-pipeline.ll

```diff
@@ -8,15 +8,15 @@
 ; RUN: env REPOFILE=%t3.db opt -mtriple="x86_64-pc-linux-gnu-repo" -O3 -debug-pass=Structure < %s -o /dev/null 2>&1 | FileCheck %s
 ; RUN: rm -f %t4.db
 ; RUN: env REPOFILE=%t4.db opt -mtriple="x86_64-pc-linux-gnu-repo" -Os -debug-pass=Structure < %s -o /dev/null 2>&1 | FileCheck %s
+; RUN: rm -f %t5.db
+; RUN: env REPOFILE=%t5.db opt -mtriple="x86_64-pc-linux-gnu-repo" -debug-pass=Structure < %s -o /dev/null 2>&1 | FileCheck %s
 
 ; REQUIRES: asserts
 
-; CHECK:      Profile summary info
-; CHECK-NEXT:   ModulePass Manager
+; CHECK:       ModulePass Manager
```

Comment:
Why did the "ModulePass Manager" line disappear?

Reply:
Did you mean `Profile summary info` line? The `ModulePass Manager` line is still checked (just changed from `CHECK-NEXT` to `CHECK`).

The reason for deleting the  `Profile summary info`  check since the pass is not included in the pass structure when using `opt` without the optimization level.

Compile:
`opt -mtriple="x86_64-pc-linux-gnu-repo" -debug-pass=Structure repo-opt-pipeline.ll -o test.o`

Output:
```
Pass Arguments:  -targetlibinfo -tti -targetpassconfig -prepo -prepo-pruning -globaldce -verify -write-bitcode
Target Library Information
Target Transform Information
Target Pass Configuration
  ModulePass Manager
    RepoMetadataGenerationPass
    RepoPruningPass
    Dead Global Elimination
    FunctionPass Manager
      Module Verifier
    Bitcode Writer
```